score-v1b1.json

```diff
@@ -117,13 +117,6 @@
             }
           }
         },
-        "properties": {
-          "description": "DEPRECATED: The properties that can be referenced in other places in the Score Specification file.",
-          "type": [
-            "object",
-            "null"
-          ]
```

Comment:
Wouldn't it be more versioning scheme compliant to mark this as `deprecated: true` instead of completely removing it?

In my opinion this should only be removed once `v1b2` is released.

Reply:
I agree that's how it should have been handled originally rather than adding "DEPRECATED" to the description. This would have been more clear and we are doing that in some other places, however since none of our clients actually consume this spec document, it would have no effect other than to leave technical debt lying around.

Reply:
So I propose removing it entirely as a pragmatic choice rather than the theoretically correct one.

Reply:
We discussed this in the score meeting today and agreed that we can remove this.